Libs/MRML/Core/vtkMRMLStorageNode.h

```diff
@@ -117,14 +117,17 @@ class VTK_MRML_EXPORT vtkMRMLStorageNode : public vtkMRMLNode
   /// Transferring: data is remote, and the transfer is working to completion
   /// TransferDone: the data is on disk and ready to be read
   /// Cancelled: the user cancelled the remote data transfer
+  /// CancelledNoData: reading was cancelled due to data not present. Can be
+  ///                  valid in case of empty nodes such as transforms or volumes
   enum
   {
     Idle,
     Pending,
     Scheduled,
     Transferring,
     TransferDone,
-    Cancelled
+    Cancelled,
+    CancelledNoData
```

Comment:
We've discussed this with @cpinter and CancelledNoData seemed like a good name but now I feel that it suggests that something is wrong. I'm wondering if SkippedNoDataAvailable, TransferSkippedNoData, TransferNotNeededEmptyData would be more clear?

@pieper what do you think?

Reply:
yes, I prefer SkippedNoData now, that you suggest it.

Reply:
OK I'll change that. Thanks!